### ISSUE TITLE
3.6.5-rc1 - .pdb copy step fails with static openthreads

### ISSUE THREAD

**Issue by solvingj** [openscenegraph/OpenSceneGraph#898]
https://github.com/openscenegraph/OpenSceneGraph/blob/master/src/OpenThreads/win32/CMakeLists.txt#L97

This block of code failed in our testing because openthreads.pdb could not be found:
```
IF(MSVC AND DYNAMIC_OPENSCENEGRAPH)
    GET_TARGET_PROPERTY(PREFIX ${LIB_NAME} PREFIX)
    IF( ${CMAKE_GENERATOR} STREQUAL "Ninja" )
	INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PREFIX}${LIB_NAME}${CMAKE_RELEASE_POSTFIX}.pdb DESTINATION ${INSTALL_BINDIR} COMPONENT libopenthreads CONFIGURATIONS Release)
	INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PREFIX}${LIB_NAME}${CMAKE_RELWITHDEBINFO_POSTFIX}.pdb DESTINATION ${INSTALL_BINDIR} COMPONENT libopenthreads CONFIGURATIONS RelWithDebInfo)
	INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PREFIX}${LIB_NAME}${CMAKE_DEBUG_POSTFIX}.pdb DESTINATION ${INSTALL_BINDIR} COMPONENT libopenthreads CONFIGURATIONS Debug)
    ELSE( ${CMAKE_GENERATOR} STREQUAL "Ninja" )
	INSTALL(FILES ${OUTPUT_BINDIR}/${PREFIX}${LIB_NAME}${CMAKE_RELWITHDEBINFO_POSTFIX}.pdb DESTINATION ${INSTALL_BINDIR} COMPONENT libopenthreads CONFIGURATIONS RelWithDebInfo)
	INSTALL(FILES ${OUTPUT_BINDIR}/${PREFIX}${LIB_NAME}${CMAKE_DEBUG_POSTFIX}.pdb DESTINATION ${INSTALL_BINDIR} COMPONENT libopenthreads CONFIGURATIONS Debug)
    ENDIF( ${CMAKE_GENERATOR} STREQUAL "Ninja" )
ENDIF(MSVC AND DYNAMIC_OPENSCENEGRAPH)
```

I haven't had time to debug fully, but I suspect the problem is that we're checking the variable `DYNAMIC_OPENSCENEGRAPH` instead of either `DYNAMIC_OPENTHREADS` or `OPENTHREADS_USER_DEFINED_DYNAMIC_OR_STATIC`.   I think the current code happens to work in cases where both `DYNAMIC_OPENSCENEGRAPH` and `DYNAMIC_OPENTHREADS` are true, which is probably a significant number of uses.  However, we use `DYNAMIC_OPENSCENEGRAPH=True` and 
`DYNAMIC_OPENTHREADS=False`.

**Comment by openscenegraph** [comment]
I merged your PR fix based on master, but I am now trying to work out how it relates to the 3.6 release as the OpenSceneGraph/src/OpenThreads/win32/CMakeLists.txt in 3.6 is very different from the one in master. The modified lines in your PR don't exist at all in 3.6.

I am now confused as to whether 3.6.5 does have the pdb copy problem, or whether it was just always master that had the problem that is now fixed.  If 3.6 still has the pdb copy problem then we'll need to modify the 3.6 branch, but it'll need to be a different fix as the modified blocks don't exist in 3.6.x.

**Comment by solvingj** [comment]
Apologies, my mistake.  Indeed this only affected master, and I got my checkouts confused between 3.6.5-rc1 and master when I opened the ticket.  I also used the Github GUI to submit the PR in a hurry so I didn't notice there either. 

Should I update the ticket title to be correct for posterity before closing?
